src/identity-user-interfaces.ts

```diff
@@ -62,6 +62,10 @@ export interface IUserIdentityChangeEvent extends BaseEvent {
     userIdentityChanges: ISDKUserIdentityChanges;
 }
 
+export interface ISDKUserAttribute {
```

Comment:
nit: I think this should be plural. saying UserAttribute makes it sound like single attribute, which could be either the key or the value.  We want to make it more clear that it is the object of all k:v pairs.

```suggestion
export interface ISDKUserAttributes {
```